Description:
 [2e49cf0a] Add another Xoom variant
 [7c1b9074] New GoGear device
Origin: upstream,
 http://libmtp.git.sf.net/git/gitweb.cgi?p=libmtp/libmtp;a=commit;h=2e49cf0a
 http://libmtp.git.sf.net/git/gitweb.cgi?p=libmtp/libmtp;a=commit;h=7c1b9074
Applied-Upstream: yes
---
 src/music-players.h |    9 +++++++++
 1 file changed, 9 insertions(+)

--- libmtp.orig/src/music-players.h
+++ libmtp/src/music-players.h
@@ -348,6 +348,9 @@
   // From Anonymous SourceForge User
   { "Philips", 0x0471, "GoGear VIBE SA2VBE[08|16]K/02", 0x20b7,
       DEVICE_FLAG_UNLOAD_DRIVER },
+  // From Anonymous SourceForge User
+  { "Philips", 0x0471, "GoGear Ariaz", 0x20b9,
+      DEVICE_FLAG_UNLOAD_DRIVER },
   // from XNJB user
   { "Philips", 0x0471, "PSA235", 0x7e01, DEVICE_FLAG_NONE },
 
@@ -1243,6 +1246,12 @@
       DEVICE_FLAG_BROKEN_MTPGETOBJPROPLIST |
       DEVICE_FLAG_BROKEN_SET_OBJECT_PROPLIST |
       DEVICE_FLAG_BROKEN_SEND_OBJECT_PROPLIST },
+  // WiFi-only version of Xoom
+  // See: http://bugzilla.gnome.org/show_bug.cgi?id=647506
+  { "Motorola", 0x18d1, "Xoom (MZ604)", 0x70a8,
+      DEVICE_FLAG_BROKEN_MTPGETOBJPROPLIST |
+      DEVICE_FLAG_BROKEN_SET_OBJECT_PROPLIST |
+      DEVICE_FLAG_BROKEN_SEND_OBJECT_PROPLIST },
   { "Motorola", 0x22b8, "Xoom (ID 2)", 0x70a9,
       DEVICE_FLAG_BROKEN_MTPGETOBJPROPLIST |
       DEVICE_FLAG_BROKEN_SET_OBJECT_PROPLIST |
